### ISSUE TITLE
Select sometimes render blank options after version 4.6.0

### ISSUE THREAD

**Issue by patrickshipe** [ant-design/ant-design#26468]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/antd-select-showing-blanks-issue-k85v9?file=/index.js)

### Steps to reproduce
1. Open Select dropdown, and select an option near the middle.
2. Open and close select dropdown via clicking the arrow. 
Every few times, options above or below the selected option will render as blank spaces.

### What is expected?
All options render.

### What is actually happening?
Some options render as blank spaces.
![image](https://user-images.githubusercontent.com/42809110/91568407-ffdf7200-e90a-11ea-84d9-460afc029b69.png)

**NOTE** this does _not_ happen locally in development environment, probably because the dev environment forces extra rerenders. It only happens in an environment built for production.

| Environment | Info |
|---|---|
| antd | 4.6.1 |
| React | 16.13.1 |
| System | macOS Catalina |
| Browser | Chrome 84 |

**Comment by xrkffgg** [comment]
Duplicate of https://github.com/ant-design/ant-design/issues/26116

**Comment by afc163** [comment]
Fixed in #26116, please remove package-lock.json and yarn.lock and reinstall node_modules.

**Comment by SonnyZoetis** [comment]
... I doubt it has been solved, I have same issue in Ant Design 5.9.4. with the Select mode="multiple" ...
Even tried removing all options, Ant still renders the empty option. Tried everything, Ant keeps adding the empty option... 

```
<Select 
  mode="multiple"
  allowClear
  defaultValue={data.value}
  options={items.map((item: any) => ({
    value: item.value,
    label: item.label,
  }))}
/>
```
